src/ray/gcs/gcs_client/service_based_accessor.cc

```diff
@@ -767,6 +767,44 @@ void ServiceBasedNodeInfoAccessor::AsyncResubscribe(bool is_pubsub_server_restar
   }
 }
 
+Status ServiceBasedNodeInfoAccessor::AsyncSetInternalConfig(
+    std::unordered_map<std::string, std::string> &config) {
+  rpc::SetInternalConfigRequest request;
+  rpc::StoredConfig cfg;
+  for (auto &pair : config) {
+    (*cfg.mutable_config())[pair.first] = pair.second;
+  }
+  request.mutable_config()->CopyFrom(cfg);
```

Comment:
I think `cfg` is not needed here, you can use the following code.
`request.mutable_config()->insert(config.begin(), config.end())`

Reply:
Oh yep! Thanks